src/NServiceBus.Core/Encryption/RijndaelEncryptionService.cs

```diff
@@ -227,6 +227,11 @@ protected virtual bool TryGetKeyIdentifierHeader(out string keyIdentifier, IInco
             return context.Headers.TryGetValue(Headers.RijndaelKeyIdentifier, out keyIdentifier);
         }
 
+        protected virtual void GenerateIV(RijndaelManaged rijndael)
```

Comment:
>  This is now only exposed internally to aid testability but could become more relevant if we break encryption out of the core and allow more customization by users.

let's focus on the testing story here (the class is still internal). I'm :+1: to @danielmarbach suggestion to pass required parameters (if we need any parameters currently?).